src/components/CircularButton.tsx

```diff
@@ -5,18 +5,22 @@ type Direction = "left" | "top" | "right";
 export interface ICircularButtonProps {
   direction?: Direction;
   size?: number;
+  className?: string;
+  handlerOnClick?: (newVal?) => void;
 }
-
 const CircularButton: FunctionComponent<ICircularButtonProps> = (props) => {

```

Comment:
Let's deconstruct property from `props` to keep consistency with other components. It will helps to prevent error when property is declared, by never used.

>const CircularButton: FunctionComponent<ICircularButtonProps> = (props) => {

 **to** 

>const CircularButton: FunctionComponent<ICircularButtonProps> = ({**direction, size, className, handlerOnClick**}) => {

Reply:
- [ ] Done [4933346](https://github.com/AnastasiSo/Portfolio-Project/pull/16/commits/49333460f0c77e2c28460b3246efbaf35b4b3ce0).